mongo/assets/configuration/spec.yaml

```diff
@@ -99,20 +99,42 @@ files:
         type: integer
         example: 30
     - name: ssl
+      deprecation:
```

Comment:
I think we can rename the options directly as long as in the code we still consume the old ones. This is what we've been doing in other integrations

Reply:
Do you mean removing them (as the new options already exist separately). FWIW this integration's config already has (and I think it's the only one to have it) an option with the `deprecation` field (added [here](https://github.com/DataDog/integrations-core/pull/6574/files#diff-30add79992152c0661aea77f6f6e6c6cca5457bdb45937f2eb76bac838b78315)). The `deprecation` option is documented [here](https://datadoghq.dev/integrations-core/meta/config-specs/#options) and [here](https://datadoghq.dev/integrations-core/meta/config-models/#deprecation). Was the idea of using it abandoned? 

Thus far I haven't found other examples of option deprecations done in the past. It looks like this may be worth some documentation improvements.